packages/components/src/components/Alert/Alert.tsx

```diff
@@ -143,8 +128,9 @@ export const Alert: FC<AlertProps> = ({
   const iconViewStyle: ViewStyle = {
     flexDirection: 'row',
     // Below keeps icon aligned with first row of text, centered, and scalable
+    // If Text variant for header changes, token referenced in minHeight must change accordingly
```

Comment:
Not sure if there's any good way to rework, but seems unlikely this comment would be noticed in that eventuality.